src/stars.css

```diff
@@ -0,0 +1,47 @@
+.container {
+  height: 160px;
+  width: 140px;
+  margin: 0;
+}
```

Comment:
What `container` block styles are doing in `stars` block file?
Also, don't add margin to blocks